cmd/crio/config.go

```diff
@@ -39,14 +39,32 @@ listen = "{{ .Listen }}"
 # runtime used and options for how to set up and manage the OCI runtime.
 [crio.runtime]
 
-# runtime is a path to the OCI runtime which crio will be using.
+# runtime is the OCI compatible runtime used for trusted containers.
+# This is a mandatory setting as this runtime will be the default one
+# and will also be used for untrusted containers if runtime_untrusted_container
+# is not set.
 runtime = "{{ .Runtime }}"
 
-# runtime_host_privileged is a path to the OCI runtime which crio
-# will be using for host privileged operations.
-# If this string is empty, crio will not try to use the "runtime"
-# for all operations.
-runtime_host_privileged = "{{ .RuntimeHostPrivileged }}"
+# runtime_untrusted_container is the OCI compatible runtime used for untrusted
+# containers. This is an optional setting, except if default_container_trust is
+# set to "untrusted".
+runtime_untrusted_container = "{{ .RuntimeUntrustedContainer }}"
+
+# default_container_trust is the default level of trust crio puts in containers.
+# This can either be "trusted" or "untrusted".
```

Comment:
Does this default to "untrusted"?  If so, I'd suggest appending " and the default is "untrusted"."